.github/workflows/ci.yml

```diff
@@ -359,4 +358,4 @@ jobs:
 
       - name: Run compiler-tester tests
         working-directory: ${{ github.workspace }}/era_vm/era-compiler-tester
-        run: ./target/release/compiler-tester --target EraVM ${{ matrix.mode }} --path ${{ matrix.testgroup }}
+        run: ./target/release/compiler-tester --target EraVM ${{ matrix.mode }} --path ${{ matrix.testgroup }}
```

Comment:
Missing new line here.

Reply:
That is because of the deleted line above

Reply:
I mean the precompile test at the beginning